sql/parser/select.go

```diff
@@ -68,10 +68,10 @@ const (
 )
 
 // Select.Lock
-const (
-	astForUpdate = " FOR UPDATE"
-	astShareMode = " LOCK IN SHARE MODE"
-)
+// const (
+// 	astForUpdate = " FOR UPDATE"
+// 	astShareMode = " LOCK IN SHARE MODE"
+// )
```

Comment:
You can just delete this. When/if we implement I likely won't use these constants.